tests/misc_object_tests/FileObject_test.py

```diff
@@ -43,7 +43,7 @@ def test_empty_file_object() -> None:
 
 
 def test_blank_file_object_filename() -> None:
-    dobj = Objects.DFXMLObject(version="1.2.0")
+    dobj = Objects.DFXMLObject(version="2.0.0-beta.0")
```

Comment:
Should we have all of these strings in a single location?

Reply:
I'll put the value into `/dfxml/__init__.py`, [next to the XMLNS constants](https://github.com/dfxml-working-group/dfxml_python/blob/68fef71b5c432e5f7c2fbbe84903ff94bd29cd6e/dfxml/__init__.py#L64-L66).

I think I'll also just have the `version` keyword argument default to that constant string.

Reply:
I want a single constant string... Should I wait off on merging this?

Reply:
Yes, please.  I was mid-stream on something else, will handle now.

Reply:
This is addressed now.  If you have no other feedback, I'm fine with this being merged at your convenience.